numpy/core/numeric.py

```diff
@@ -1,8 +1,10 @@
 def __getattr__(attr_name):
     from numpy._core import numeric
     from ._utils import _raise_warning
-    ret = getattr(numeric, attr_name, None)
-    if ret is None and attr_name != "newaxis":
+
+    sentinel = object()
```

Comment:
Not really this PR, but I don't think I care too much.

Reply:
If I don't do this, test fails because `_no_nan_dtypes` is `None`.